core/src/windows/cpp/CppPerfCounters.vcxproj

```diff
@@ -48,28 +48,28 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'" Label="Configuration">
     <ConfigurationType>DynamicLibrary</ConfigurationType>
     <UseDebugLibraries>true</UseDebugLibraries>
-    <PlatformToolset>v120</PlatformToolset>
+    <PlatformToolset>v141</PlatformToolset>
```

Comment:
v120 is an ancient visual studio 2013 runtime. It's not something that'd be typically recommended in 2019.

v141 is Visual Studio 2015+ runtime, compatible with 2015/2017+ runtime .dll

Reply:
Just checked that the latest JDK11 uses v141. The current implementation depends on both .NET 4.6+ and vc redist to be deployed on machine. Ideally at some point we'd need to get rid of C++/CLI dependency and use Win32 API directly to avoid the bloat.